rpm2cpio.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: rpm2cpio
+  version: 14.2.0
+  epoch: 0
+  description: Convert .rpm files to cpio format
+  copyright:
+    - license: BSD-2-Clause
+  dependencies:
+    runtime:
+      - libarchive-tools
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://raw.githubusercontent.com/freebsd/freebsd-ports/refs/heads/main/archivers/rpm2cpio/files/rpm2cpio
+      expected-sha256: 2841bacdadde2a9225ca387c52259d6007762815468f621253ebb537d6636a00
+      extract: false
+
+  - runs: |
+      install -Dm755 rpm2cpio ${{targets.destdir}}/usr/bin/rpm2cpio
+      sed -i 's/tar/bsdtar/g' ${{targets.destdir}}/usr/bin/rpm2cpio
+
+update:
+  enabled: false
+  exclude-reason: single script file which hasn't been updated in last 10 years.
+
+test:
+  pipeline:
+    - runs: |
+        rpm2cpio
```

Comment:
We should be able to fetch any RPM and then use cpio to parse

Reply:
instead fetching the yq rpm from an internet which may be removed in the future don't you think that we could add this file into the package folder? but other than that this PR looks so cool!

Reply:
I had this fear :homerdisappear: 

but the rpm size looks not so big so I think we can put it inside the package directory, I'll do that in the subsequent commit. 

```bash
 $ ls -lah yq.rpm
-rw-r--r--    1 user     docker    307.7K Feb 12 07:03 yq.rpm
```
I also tried if we can package a `*.rpm` package easily with some dummy files and turns out that's not easy (phew). :(

Reply:
It seems you've added test case! Resolving this discussion.